internal/services/monitor/client/client.go

```diff
@@ -7,6 +7,7 @@ import (
 	classic "github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2021-07-01-preview/insights"
 	newActionGroupClient "github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2021-09-01-preview/insights"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/common"
+	newAlertsManagementClient "github.com/hashicorp/terraform-provider-azurerm/internal/services/monitor/sdk/2021-08-08/alertsmanagement"
```

Comment:
can we move as many resources over to the newest SDKs as possible?

Reply:
The [2021-08-08](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2021-08-08) version of REST API specification has only [AlertProcessingRules](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2021-08-08/AlertProcessingRules.json) resource under it. Maybe we should change the import alias from `newAlertsManagementClient` to `AlertProcessingRulesClient`:
```suggestion
	AlertProcessingRulesClient "github.com/hashicorp/terraform-provider-azurerm/internal/services/monitor/sdk/2021-08-08/alertsmanagement"
```

And moving other resources of [AlertsManagement ](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement)to newest SDKs could be done in upcoming PRs.

Reply:
will change

Reply:
Hi @katbyte, thank you for reviewing my code. This is a new resource developed with Pandora SDK which is generated from this [swagger path](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2021-08-08). Since it is the only resource defined in this [swagger path](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2021-08-08), maybe moving existing resources to this newer SDK does not apply here? Or do I need to update other resources to Pandora SDK ?